ic-http/src/http_request.rs

```diff
@@ -15,5 +15,5 @@ pub async fn http_request(arg: CanisterHttpRequestArgument) -> CallResult<(HttpR
 /// Make an HTTP request to a given URL and return the HTTP response, possibly after a transformation.
 #[cfg(target_arch = "wasm32")]
 pub async fn http_request(arg: CanisterHttpRequestArgument) -> CallResult<(HttpResponse,)> {
-    ic_cdk::api::management_canister::http_request::http_request(arg).await
+    ic_cdk::api::management_canister::http_request::http_request_with_cycles(arg, 0).await
```

Comment:
Isn't this crate meant to be general-purpose? If yes, then this won't work for anyone using this library in canister that's running on an app subnet.

Reply:
Right, fixed, thank you!